lib/types.ts

```diff
@@ -58,10 +58,8 @@ export interface HudParams {
 }
 
 export interface PRData {
-  sha: string;
-  eventTime: string;
-  commitTitle: string;
   title: string;
```

Comment:
Do we need some other data here? Looks it also headSha and headCommitTitle. Not sure if we use that

Reply:
it's duplicative with the `shas` field, so I'll just remove it from the rockset query. Eventually we may want to include other stuff through (like PR body as discussed)